scripts/travis_ubuntu.sh

```diff
@@ -3,10 +3,17 @@
 ## -x be verbose; write what we are doing, as we do it
 set -ex
 
+cmd=ls
+if [ $3 = "old" ]; then
```

Comment:
Or if no other solution, killing old and replacing with `if [ $2 = "eoan" ]; then ` ? What do you think ?

Reply:
We may encounter EOLs of many non-LTS ubuntu systems in the future. Thus I use the `old` option to tell the script that this is an EOL system so it will try to subsititute the mirror address. I can remove it if you think `if [ $2 = "eoan" ]` way is better.

Reply:
I have no idea how to get ubuntu eoan to work without changing `sources.list`. I've removed the `old` option and replaced with `if [ $2 = ":eoan" ]; then`.